mods/beds/functions.lua

```diff
@@ -81,6 +81,21 @@ local function lay_down(player, pos, bed_pos, state, skip)
 
 	-- lay down
 	else
+
+		-- Check if bed is occupied
+		for _, other_pos in pairs(beds.bed_position) do
+			if vector.distance(bed_pos, other_pos) < 0.1 then
+				minetest.chat_send_player(name, S("This bed is already occupied!"))
```

Comment:
This is only for information. Full stop rather than exclamation mark would be rather appropriate IMO.

Reply:
Do you think a full stop would do, or simply nothing?

Reply:
Done